### PR TITLE
Play 2.8 and Scala 2.13 upgrade

### PR DESCRIPTION
**Pull request by aramlaka**
Upgrades apibuilder to Play 2.8 and Scala 2.13, but you can read the title

### CODE REVIEW

**Comment by aramlaka** [comment]
`No implementation for io.flow.util.Config was bound.` Error in db test cases; not sure how to begin fixing

File: api/conf/base.conf [hunk 1]
```diff
@@ -17,6 +17,7 @@ play.http.parser.maxMemoryBuffer=10M
 play.http.requestHandler = "play.http.DefaultHttpRequestHandler"
 
 play.modules.enabled += "actors.ActorsModule"
+play.modules.enabled += "io.flow.play.clients.ConfigModule"
```
Comment (aramlaka) [inline api/conf/base.conf]: :octocat:

File: swagger/src/test/scala/me/apidoc/swagger/SwaggerServiceValidatorSpec.scala [hunk 2]
```diff
@@ -111,6 +101,16 @@ class SwaggerServiceValidatorSpec extends FunSpec with Matchers {
                       `type` = "long",
                       required = true
                     ),
+                    Field(
```
Comment (juwen-z) [inline swagger/src/test/scala/me/apidoc/swagger/SwaggerServiceValidatorSpec.scala]: Should we keep the original order in `json` ?
Would it be an unexpected behavior if generated code have different order than originally specified in `json`?

Comment (benwaffle) [reply swagger/src/test/scala/me/apidoc/swagger/SwaggerServiceValidatorSpec.scala]: objects in JSON are unordered, so the original code is not any better than what I have here

Model.fields is a Seq, but it should really be a Set. I don't think the fix is trivial, so I just wanted the tests to pass for now.

**Pull request merged**